taxcalc/growth.json

```diff
@@ -0,0 +1,24 @@
+{
+    "_factor_adjustment":{
+        "long_name": "Deviation from CBO forecast of baseline economic growth (percentage point",
```

Comment:
the _factor_target allows the user to specify a GDP growth rate as a percent. The _factor_adjustment allows the user to specify a percentage point deviation from existing rates (If the existing growth rate is 10%, a 10 percent increase would make it 11%, a 10 percentage point increase would make it 20%.)